models/tmp/stg_apple_store__app_store_device_tmp.sql

```diff
@@ -1 +1,2 @@
-select * from {{ var('app_store_device') }}
+select * 
+from {{ var('app_store_device') }}
```

Comment:
I'll push a change into the staging model automation in our automations package to account for this so we don't have to manually do it in the future 😅